rsfbclient-native/src/ibase.rs

```diff
@@ -650,9 +650,9 @@ parse_functions! {
     //         arg9: *mut ISC_SCHAR,
     //     ) -> ISC_STATUS;
     // }
-    // extern "C" {
-    //     pub fn isc_vax_integer(arg1: *const ISC_SCHAR, arg2: ::std::os::raw::c_short) -> ISC_LONG;
-    // }
+    extern "C" {
+        pub fn isc_vax_integer(arg1: *const ISC_SCHAR, arg2: ::std::os::raw::c_short) -> ISC_LONG;
+    }
```

Comment:
```suggestion
//    extern "C" {
//        pub fn isc_vax_integer(arg1: *const ISC_SCHAR, arg2: ::std::os::raw::c_short) -> ISC_LONG;
//    }
```
1. does not seem to be used anywhere
2. i think it's deprecated in favor of `isc_portable_integer` ([link](https://firebird-devel.narkive.com/cmZiIKZI/extending-precision-of-api-information-values)). this is definitely the case for interbase ([link](http://docwiki.embarcadero.com/InterBase/2020/en/Isc_vax_integer()))